pylintrc~

```diff
@@ -0,0 +1,285 @@
+[MASTER]
```

Comment:
whoops, don't forget to revert this file, it's a temp file created by your editor.